server/data/localMockData/locations.ts

```diff
@@ -0,0 +1,58 @@
+type Location = {
```

Comment:
I think this is the wrong model - should be `CaseLoad` which represents a prison.
e.g.
```
  {
    "caseLoadId": "MDI",
    "description": "Moorland Closed (HMP & YOI)",
    "type": "INST",
    "caseloadFunction": "GENERAL",
    "currentlyActive": false
  }
```